Fix #10856: Sync billing with shipping address on Admin Order Page

YevSent opened the pull request:
### Fixed Issues
1. magento/magento2#10856: Sync billing with shipping address on Admin Reorder and Admin Customer Create Order page does not work for Existing address selected

### Contribution checklist
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds on Travis CI are green)

YevSent committed:
Sync billing with shipping address on Admin Order Page

 - Set customer address ID only if the address was selected in dropdown
 - Removed "old" billing address from quote if customer address doesn't specified

magento-cicd2 commented:
[![CLA assistant check](https://cla.dev.magento.com/pull/badge/not_signed)](https://cla.dev.magento.com/magento/magento2?pullRequest=11385) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/magento2?pullRequest=11385) before we can accept your contribution.<br/>

YevSent committed:
Sync billing with shipping address on Admin Order Page

 - Fixed failed FAT

YevSent committed:
Sync billing with shipping address on Admin Order Page

 - Added validation for empty applied taxes to avoid exceptions during json decoding

joni-jones commented:
@magento-team, I see `needs update` label, any questions about this PR?

YevSent committed:
Sync billing with shipping address on Admin Order Page

 - Fixed skipped 'Same as billing' checkbox
